lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -513,12 +513,12 @@ public static Domain getDomain(
             return Domain.all(type);
         }
 
-        long totalNullCount = nullCounts.stream()
+        Optional<Long> totalNullCount = nullCounts.map(pageNullCounts -> pageNullCounts.stream()
                 .mapToLong(value -> value)
-                .sum();
-        boolean hasNullValue = totalNullCount > 0;
+                .sum());
```

Comment:
nit. `totalNullCount` could still be `long` if we just do `orElse(0L)` then `boolean hasNullValue = totalNullCount> 0;` could stay without change.

Reply:
While that would be correct in the current code, I'm still preferring to keep the optional as it guards against the possibility of someone misusing the `orElse(0L)` value in future.